keyboards/kinesis/kintwin/info.json

```
@@ -0,0 +1,239 @@
+{
+    "processor": "STM32F401",
+    "bootloader": "stm32-dfu",
+    "board": "BLACKPILL_STM32_F401",
+    "keyboard_name": "Kinesis",
+    "url": "https://github.com/alvicstep/kintwin",
+    "manufacturer": "https://github.com/alvicstep/kintwin",
+    "maintainer": "Alex Stepanov",
+    "usb": {
+        "device_version": "0.0.3",
+        "vid": "0xFEED",
+        "pid": "0x4B4E"        
+    },
```

migrated from `rules.mk`
```suggestion
    "usb": {
        "device_version": "0.0.3",
        "vid": "0xFEED",
        "pid": "0x4B4E",
        "shared_endpoint": {
            "keyboard": true
        }
    },
    "features": {
        "bootmagic": true,
        "console": false,
        "command": false,
        "mousekey": false,
        "extrakey": false,
        "nkro": true,
    },
```